employee-rostering-gwtui/src/main/java/org/optaweb/employeerostering/gwtui/client/common/StringListToSkillSetConverter.java

```diff
@@ -103,17 +108,14 @@ private void updateSkillMappings(List<Skill> skillList) {
         for (Skill skill : skillList) {
             skillMap.put(skill.getName(), skill);
         }
-        MessageBuilder.createMessage()
-                .toSubject("SkillMapListener")
-                .withValue(getSkillMap())
-                .noErrorHandling().sendNowWith(ErraiBus.getDispatcher());
+        eventManager.fireEvent(Event.SKILL_MAP_INVALIDATION, skillMap);
     }
 
     private Promise<List<Skill>> getSkillList() {
         return new Promise<>((resolve, reject) -> SkillRestServiceBuilder.getSkillList(tenantStore.getCurrentTenantId(), FailureShownRestCallback
-                .onSuccess(newSkillList -> {
-                    resolve.onInvoke(newSkillList);
-                })));
+                                                                                                                                                 .onSuccess(newSkillList -> {
+                                                                                                                                                     resolve.onInvoke(newSkillList);
+                                                                                                                                                 })));
```

Comment:
@yurloc @rsynek Are any of you guys a fan of the way method parameters get idented by the general kie style?

Reply:
I am not, to be honest

Reply:
@ge0ffrey you could not have picked a better illustration to get the answer "No!" :) I even wonder why we have such a thing as KIE code style. Why not simply taking Oracle Java conventions? In contrast to KIE code style, it's already available for every existing IDE and maintenance cost for the team is none.

Reply:
If this is formatted with KIE Java Conventions then the Eclipse version of the formatting rules is broken. IDEA doesn't do this. Please do not commit this.

Also, this is not related to parameter indentation (in this case, all parameters are on the same line).